doc/api/esm.md

```diff
@@ -1517,6 +1518,18 @@ loaded from disk but before Node.js executes it; and so on for any `.coffee`,
 `.litcoffee` or `.coffee.md` files referenced via `import` statements of any
 loaded file.
 
+You could also chain this loader together with the HTTPS loader from above:
+
+```coffee
+# remote.coffee
+import foo from 'https://example.org/resource.coffee'
+console.log foo
```

Comment:
Can this please use the example from the https loader, with `import { VERSION }`? Then the example would be runnable.

Reply:
as I said before, the point is that it loads coffeescript over https.

Reply:
This example is a bit confusing to me because it's using the CoffeeScript loader to do two things: transpile a local file, _and_ transpile the source passed to it from the HTTPS loader. It would almost be clearer if the local file were JS, to emphasize that the two loaders are both operating on the remote file. I'm not saying that that's a change we should make, but the current example feels off somehow.

This example is almost a CoffeeScript parallel to Deno's pattern of TypeScript locally and `import` of remote TypeScript, so maybe it's fine as is.